docs/views/about.html

```diff
@@ -0,0 +1,75 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+About - GOV.UK Prototype kit
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="breadcrumbs">
+        <ol role="breadcrumbs">
+          <li><a href="/docs">GOV.UK prototype kit</a></li>
+        </ol>
+      </div>
+      <h1 class="heading-xlarge">About</h1>
+
+      <p class="lede">
+        The prototype kit provides a simple way to make interactive prototypes that look like pages on GOV.UK. These prototypes can be used to show ideas to people you work with, and to do user research
+      </p>
+
+      
+      <h2 class="heading-medium">Principles</h2>
+
```

Comment:
Think we need to lead with:

'The prototype kit:'

Otherwise, all the bullets are sentence fragments.